superset/assets/version_info.json

```diff
@@ -1 +1 @@
-{"GIT_SHA": "2d08e240285288b71df98747ddd4b6cca3220c5a", "version": "0.15.2"}
\ No newline at end of file
+{"GIT_SHA": "1e94498d9d548cbea6466a45dafa3b919c65bd1f", "version": "0.15.4"}
```

Comment:
we should `git rm --cached` this file and add it to the .gitignore list... (my bad)